.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Build and Push Docker Images
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository_owner }}
```

Comment:
Presently the images are named "rundetection": https://github.com/orgs/interactivereduction/packages/container/package/rundetection

Reply:
The Image name var just sets the `interactivereduction` part of the name (I basically stole this from jobcontroller). I have removed the hyphen in the below part though.

Reply:
Ah ok both of these work then